content/ops/managing-users.md

```diff
@@ -5,6 +5,12 @@ menu:
 title: Managing Users
 ---
 
+## Note for GovCloud environment
```

Comment:
@brittag good point, this is a note for administrators though and they should know which environment they are targeting. I was thinking of removing invite.cloud.gov from the GovCloud environment as a way to protect the users.